nexus/external-api/output/nexus_tags.txt

```diff
@@ -65,6 +65,10 @@ support_bundle_list                      GET      /experimental/v1/system/suppor
 support_bundle_view                      GET      /experimental/v1/system/support-bundles/{bundle_id}
 system_update_get_repository             GET      /v1/system/update/repository/{system_version}
 system_update_put_repository             PUT      /v1/system/update/repository
+system_update_trust_root_create          POST     /v1/system/update/trust-roots
```

Comment:
Could you file an issue for moving all the update-related ones out of "experimental" by R17?

Reply:
https://github.com/oxidecomputer/omicron/issues/8562